dallinger/command_line.py

```diff
@@ -334,7 +334,18 @@ def debug(verbose, bot, exp_config=None):
             # Call endpoint to launch the experiment
             log("Launching the experiment...")
             time.sleep(4)
-            requests.post('{}/launch'.format(base_url))
+            launch_request = requests.post('{}/launch'.format(base_url))
+            try:
+                launch_data = launch_request.json()
+            except ValueError:
+                error("Error parsing response from /launch: {}".format(launch_request.content))
+                raise
+
+            if not launch_request.ok:
+                error('Experiment launch failed, check web dyno logs for details.')
+                if launch_data.get('message'):
+                    error(str(launch_data['message']))
```

Comment:
Can we be more specific about what's happening here, rather than coercing to a string? Either string formatting if the message isn't stringy, or encoding if it's a unicode string?